client/my-sites/patterns/hooks/use-render-readymade-template.ts

```diff
@@ -1,23 +1,41 @@
-import { RenderedPattern } from '@automattic/block-renderer';
 import { useQuery } from '@tanstack/react-query';
 import wpcom from 'calypso/lib/wp';
 import { RENDERER_SITE_ID } from 'calypso/my-sites/patterns/constants';
 import type { ReadymadeTemplate } from 'calypso/my-sites/patterns/types';
 
+// Shouldnt have to do this.
+function makeSlug( text: string ) {
+	return text
+		.trim()
+		.toLowerCase()
+		.replace( /[^a-z0-9]+/g, '-' )
+		.replace( /^-+|-+$/g, '' ); // remove runs of dashes and trailing dashes
+}
+
 export const useRenderReadymadeTemplate = ( readymadeTemplate: ReadymadeTemplate ) =>
-	useQuery< { [ key: string ]: RenderedPattern } >( {
+	useQuery( {
 		queryKey: [ 'pattern-library', 'readymade-template', readymadeTemplate.template_id, 'render' ],
 		queryFn() {
-			return wpcom.req.get(
+			const x = wpcom.req.post(
 				{
-					path: `/sites/${ RENDERER_SITE_ID }/block-renderer/patterns/render`,
+					path: `/sites/${ RENDERER_SITE_ID }/block-renderer/content/render`,
 					apiNamespace: 'wpcom/v2',
+					method: 'POST',
 				},
 				{
-					pattern_ids: readymadeTemplate.patterns.map( ( { id } ) => id ).join( ',' ),
 					site_title: readymadeTemplate.title,
```

Comment:
This site title plan seems to be ignored. All the previews are displaying "Assembler" as site title.

Reply:
Nice catch, @mmtr! It was a problem in the API. I've updated the diff and now the site name is working properly.